ui/src/styles/prose.css

```
@@ -105,7 +105,7 @@
   }
 
   .prose-markdown h1 {
-    font-family: 'Fraunces', serif;
+    font-family: 'Syne', sans-serif;
```

Stylelint (17.6.0)

[error] Unexpected quotes around "Syne" (font-family-name-quotes)

(font-family-name-quotes)